Artemis/Artemis/DeviceProviders/Arduino/ArduinoStrip.ino

```diff
@@ -0,0 +1,32 @@
+#include "FastLED.h"//https://github.com/FastLED/FastLED
+FASTLED_USING_NAMESPACE
+#if defined(FASTLED_VERSION) && (FASTLED_VERSION < 3001000)
+#warning "Requires FastLED 3.1 or later; check github for latest code."
+#endif
+
```

Comment:
Hard-coded values here somewhat prevent anyone from using it unless they have a very specific secondary hardware setup. If there was a way to make this variable that would be ideal.

Reply:
FastLED is a lib afaik so it's just a matter of making sure you've got the right version. Perhaps an example Arduino project on the Wiki could be useful

Reply:
@SpoinkyNL I pulled this from the example. The version it is checking for is 2 years old anyways. I put it just to make sure that the library is loaded.

Reply:
Yeah, seems reasonable to me